src/wallet/spend.cpp

```diff
@@ -778,7 +778,7 @@ static bool CreateTransactionInternal(
     }
     else if ((unsigned int)nChangePosInOut > txNew.vout.size())
     {
-        error = _("Change index out of range");
+        error = _("Transaction change output index out of range");
```

Comment:
Is there any value in translating this? I mean it is not possible to set the change position via the GUI, no?